src/index.html

```diff
@@ -12,11 +12,54 @@
+                        <li class="d-inline"><a class="nav-link h-txt9 h-orange" href="mailto:biuro@aurator.com.pl">BIURO@AURATOR.COM.PL</a></li>
+                    </ul>
+                    <a class="nav-link nav-menu__lang h-txt9 d-none d-lg-inline-block" href="#">PL <img class="dropdown-img" src="img/dropdown.png" alt="dropdown-img"></a>
+                    <a class="menu-toggle menu-toggle--open d-inline-block d-lg-none" href="#"><img src="img/nav__menu-toggle.png" alt="menu-toggle"></a>
+                    <a class="menu-toggle menu-toggle--close d-none" href="#"><img src="img/nav__menu-toggle--close.png" alt="X"></a>
+                </div>
+            </div>
+            <div class="nav-mobile d-block d-lg-none">
+                <ul class="nav-mobile__list">
+                    <li class="nav-mobile__list-item"><a class="nav-link nav-link--mobile h-txt24" href="#">NAPRAWA I REGENERACJA</a></li>
+                    <li class="nav-mobile__list-item"><a class="nav-link nav-link--mobile h-txt24" href="#">PRODUKCJA</a></li>
+                    <li class="nav-mobile__list-item"><a class="nav-link nav-link--mobile h-txt24" href="#">CENTRUM NAPRAW AWARYJNYCH LOKOMOTYW</a></li>
+                    <li class="nav-mobile__list-item"><a class="nav-link nav-link--mobile h-txt24" href="#">O FIRMIE</a></li>
+                    <li class="nav-mobile__list-item"><a class="nav-link nav-link--mobile h-txt24" href="#">KONTAKT</a></li>
+                </ul>
+                <ul class="nav-mobile__list nav-mobile__list--inline d-flex d-sm-none row">
+                    <li class="nav-mobile__list-item col-5"><a class="nav-link nav-link--mobile-kontakt h-orange" href="tel:+48323012097">+32 301 20 97</a></li>
+                    <li class="nav-mobile__list-item col-7"><a class="nav-link nav-link--mobile-kontakt h-orange" href="mailto:biuro@aurator.com.pl">BIURO@AURATOR.COM.PL</a></li>
+                </ul>
+                <div class="nav-mobile__lang">
+                    <a class="nav-link nav-link--mobile h-txt24" href="#">PL <img class="dropdown-img" src="img/dropdown.png" alt="dropdown"></a>
+                </div>
+            </div>
+            <div class="banner">
+                <div class="banner__texts">
+                    <h1 class="h-h1 h-white ">NAPRAWA<br>MASZYN TRAKCYJNYCH<br>LEPSZA NIŻ INNE</h1>
+                    <div class="banner-buttons row">
+                        <a class="banner-link banner-link--with-logo h-txt10 h-white h-border-bottom--orange col-sm-6 col-lg-4" href="#">SPRAWDŻ NASZA OFERTĘ</a>
+                        <a class="banner-link h-txt10 h-white h-border-bottom col-sm-4 col-lg-3" href="#">UMÓW SIE NA ROZMOWĘ</a>
+                        <a class="banner-link h-txt10 h-white h-border-bottom--thin-gray col-5 col-sm-3 col-lg-2" href="#">ZOBACZ FILM</a>
+                    </div>
+                </div>
```

Comment:
All texts should be in normal case and we should do uppercase with css
- [ ] text should be normailzed

Reply:
put on hold